src/appleseed.python/bindblendermeshconverter.cpp

```diff
@@ -118,12 +117,10 @@ void convert_bl_mesh(
         }
     }
 
-    int uv_index = 0;
-
     // Push triangles
-    for (size_t tri_index = 0; tri_index < triangle_length; ++tri_index)
+    for (size_t tri_index = 0; tri_index < bl_faces_count; ++tri_index)
```

Comment:
For consistency's sake, rename `tri_index` to `face_index` (like you've done in other places).